config-example.js

```diff
@@ -0,0 +1,26 @@
+var Config = {
+    munisense: {
+        vendor: "",
+        secret: "",
+        gateway_ip: ""
+    },
+    local: {
+        devices: {
+            "boiler": {
```

Comment:
I would've expected devices to be an array. It feel strange to make it an object and having the keys be an arbitrary name.

Please change to something like this:

devices: [
{
ip: "",
eui64: "",
name: ""
},
{
}
]

and name is optional ofcourse.